pyproject.toml

```diff
@@ -27,4 +44,5 @@ build-backend = "poetry.core.masonry.api"
 [tool.black]
 line-length = 88
 target-version = ['py39']
-include = '\.pyi?$'
\ No newline at end of file
+include = '\.pyi?$'
+
```

Comment:
You might want to add these lines:

```toml
[tool.isort]
profile = "black"
```

It somehow makes `isort` [play well with `black`](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html).